### ISSUE TITLE
LinearRegression 코드 관련하여 질문있습니다.

### ISSUE THREAD

**Issue by roamgom** [kaggler-tv/dku-kaggle-class#8]
지난 수업과 이번 수업자료를 참고하여

교수님께서 진행한 Linear Regression 코드를 여러 column에 적용시켜 분석해보고 있습니다.

<img width="660" alt="스크린샷 2020-09-11 오후 4 33 04" src="https://user-images.githubusercontent.com/24652225/92884084-7b353d80-f44c-11ea-83d0-88c98348262f.png">

수업시간에서 진행해주신 `u`, `dered_u` column 외 correlation값이 어느정도 높은 값들로 모델을 학습시켜보는데

> `u` & `dered_u`
<img width="732" alt="스크린샷 2020-09-11 오후 4 32 49" src="https://user-images.githubusercontent.com/24652225/92884040-6f497b80-f44c-11ea-8c0c-bcda07faf44b.png">

> `g` & `dered_g`
<img width="883" alt="스크린샷 2020-09-11 오후 4 35 14" src="https://user-images.githubusercontent.com/24652225/92884394-c64f5080-f44c-11ea-9d3b-a727027ab511.png">

column 들 간의 correlation이 너무 낮거나(.7x) 너무 높으면(.98 이상) RMSE 값과 plot 출력 부분에서 값이 매우 높거나 낮게 출력됩니다.

반면, `sklearn`의 `LinearRegression`을 사용해 같은 column 값을 적용했을 때는 correlation 값과 어느정도 일관성 있게 나왔습니다.

<img width="673" alt="스크린샷 2020-09-11 오후 4 39 27" src="https://user-images.githubusercontent.com/24652225/92885038-5e4d3a00-f44d-11ea-9859-023eab9be33f.png">

수업시간에서 배운 LinearRegression 과 `sklearn`의 `LinearRegression`의 수식 형태는 동일한 것 같은데 혹시 loss function(RMSE)에 차이가 있어 결과가 다르게 나오는 것인가요?

질문에 잘못된 부분 지적해주시면 감사하겠습니다 😀
